pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -225,6 +226,14 @@ func (c *Controller) startController(tc *corev1a1.FederatedTypeConfig) error {
 		return fmt.Errorf("Error starting sync controller for %q: %v", kind, err)
 	}
 	glog.Infof("Started sync controller for %q", kind)
+	if tc.GetEnableStatus() {
```

Comment:
Should it be possible to stop a status controller without restarting the controller manager, as is possible for sync controllers?

Given that this is now the 3rd (sync and #366 being 1 and 2) controller whose state depends on `FederatedTypeConfig`, it probably makes sense for future work to target either a more extensible controller or a framework that simplifies creating new ones.  

cc: @irfanurrehman

Reply:
> Should it be possible to stop a status controller without restarting the controller manager, as is possible for sync controllers?

It is possible to stop status controller without restarting the controller manager similar to sync controller.

> Given that this is now the 3rd (sync and #366 being 1 and 2) controller whose state depends on FederatedTypeConfig, it probably makes sense for future work to target either a more extensible controller or a framework that simplifies creating new ones.

Agree.

Reply:
I think the status controller can only be stopped by stopping propagation also (i.e. it's not possible to stop status collection independently).  I think it's also concerning that failure to start the status controller will also stop the sync controller.  This suggests that a separate stop channel be used to control each status controller.